lib/auth/auth_test.go

```diff
@@ -620,3 +620,62 @@ func (s *AuthSuite) TestCreateAndUpdateUser(c *C) {
 	err = s.a.UpdateUser(ctx, user)
 	c.Assert(err, IsNil)
 }
+
+func (s *AuthSuite) TestTrustedClusterCRUDEventEmitted(c *C) {
+	s.a.IAuditLog = s.mockedAuditLog
+
+	// set up existing cluster to bypass switch cases that
+	// makes a network request when creating new clusters
+	tc, err := services.NewTrustedCluster("test", services.TrustedClusterSpecV2{
+		Enabled:              true,
+		Roles:                []string{"a"},
+		ReverseTunnelAddress: "b",
+	})
+	c.Assert(err, IsNil)
+	_, err = s.a.Presence.UpsertTrustedCluster(tc)
+	c.Assert(err, IsNil)
+
+	c.Assert(s.a.UpsertCertAuthority(suite.NewTestCA(services.UserCA, "test")), IsNil)
+	c.Assert(s.a.UpsertCertAuthority(suite.NewTestCA(services.HostCA, "test")), IsNil)
+
+	err = s.a.createReverseTunnel(tc)
+	c.Assert(err, IsNil)
+
+	createEventEmitted := false
+	s.mockedAuditLog.MockEmitAuditEvent = func(event events.Event, fields events.EventFields) error {
+		createEventEmitted = true
+		c.Assert(event, DeepEquals, events.TrustedClusterCreate)
+		return nil
+	}
+
+	// test create event for switch case: when tc exists but enabled is false
+	tc.SetEnabled(false)
+	c.Assert(tc, NotNil)
+	c.Assert(err, IsNil)
```

Comment:
`tc` is not likely to be nil here and there's no new `err` to check.
I think these two asserts are not necessary.

Reply:
agh thanks for catching, I forgot to remove these checks. I was originally calling a function before. 

removed